app/models/vote.rb

```diff
@@ -6,6 +6,8 @@ class Vote < ApplicationRecord
 
   has_paper_trail ignore: [:updated_at], meta: { conference_id: :conference_id }
 
+  validates :event, uniqueness: { scope: :user }
```

Comment:
I would write this the other way arround:

```ruby
validates :user, uniqueness: { scope: :event }
```

The error message is slightly different. :smile: